examples/fabric-admin/scripts/stop_fabric_source.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+FABRIC_ADMIN_PATH="./fabric-admin"
```

Comment:
are these paths still correct if the paths above use `out/debug` or other paths?

Should we grep for `/fabric-bridge-app` without the prefix?

Reply:
it is no more valid if we use find binary

Reply:
The issue I have is that this is misleading easily because of the assumption of `.` being the current directory. It is not and you regex on `./fabric-admin` as `<ANY_CHAR>/fabric_admin` so this matches even `some/dir/here/fabric-admin` (because `.` matches `e` in the example).

I am suggesting to just do `/fabric-admin` and `/fabric-bridge-app` (since I am unclear why we want the . there .... is there any benefit of having a any-char-grep in there?)